Sources/MessagingInApp/Views/UIKitInline.swift

```diff
@@ -243,28 +249,7 @@ public class InAppMessageView: UIView {
 
         stopShowingMessageAndCleanup()
 
-        animateHeight(to: 0)
-    }
-
-    private func animateHeight(to height: CGFloat) {
-        // this function can be called multiple times in short period of time so we could be in the middle of 1 animation. Cancel the current one and start new.
-        runningHeightChangeAnimation?.stopAnimation(true)
-
-        runningHeightChangeAnimation = UIViewPropertyAnimator(duration: 0.3, curve: .easeIn, animations: {
-            self.heightConstraint?.constant = height // Changing the height in animation block indicates we want to animate the height change.
-
-            // Since we modified constraint, perform a UI refresh to apply the change.
-            // It's important that we call layoutIfNeeded on the topmost superview in the hierarchy. During development, there were animiation issues if layoutIfNeeded was called on a different superview then the root.
-            // Example, given this UI:
-            // UIViewController
-            // └── UIStackView
-            //    └── InAppMessageView
-            // ...If we call layoutIfNeeded on superview (UIStackView), the animation will not work as expected.
-            // This is also why it's important that we do QA testing on the inline View when it's nested in a UIStackView.
-            self.getRootSuperview()?.layoutIfNeeded()
-        })
-
-        runningHeightChangeAnimation?.startAnimation()
```

Comment:
As the PR description mentioned, I am proposing a small architecture change where UIKit and SwiftUI have their own View that performs height animations. 

This means that this existing UIKit inline View will have the height animation code moved into a separate UIKit class (I'll do this in a follow-up PR). I deleted the animation code in this PR because it was making the SwiftUI inline View not work. 

Instead, replace all height animation changes to simply sending the height change event to the `onSizeChangedDelegate` delegate.

Reply:
I do not see test cases being added/updated. Do you plan to write test cases for these updates in another PR ?

Reply:
Good question. I plan to add tests in a follow-up PR since some refactors will need to be done to get UIKit solution working again.